crates/prover/src/runner.rs

```diff
@@ -202,85 +299,136 @@ where
             .get_prover_last_scanned_l1_height()
             .unwrap_or_else(|_| panic!("Failed to get last scanned l1 height from the ledger db"));
 
-        let mut l1_height = match last_scanned_l1_height {
-            Some(height) => height.0 + 1,
+        let start_l1_height = match last_scanned_l1_height {
+            Some(height) => height.0,
```

Comment:
should be +1, no?

Reply:
no need, I increment the for loop starting range by one, otherwise things get complicated and I need to pass ledger db inside l1_sync, something I can do but unnecessary overhead
This works fine btw no extra proving whatsoever